### PR TITLE
Update meilisearch.md

### PR DESCRIPTION
**Pull request by pavlotsyhanok**
Problem:

The existing documentation for the Meilisearch plugin contains example that, if copy-pasted exactly, will not work.

Solution:

I have updated the Meilisearch plugin configuration example to reflect the correct settings. 

Benefits:

With this updated code snippet, users can simply copy and paste it into their local medusa-config.js file. It will work seamlessly with the standard configuration and is compatible with the latest version of Meilisearch.

Testing:

The updated configuration has been tested and confirmed to work in a live deployment environment on Digital Ocean(Server) and Vercel(Storefront)

image

image

### CODE REVIEW

File: www/apps/docs/content/plugins/search/meilisearch.md [hunk 1]
```diff
@@ -127,25 +127,32 @@ const plugins = [
...
-              "title", 
-              "description", 
-              "variant_sku", 
-              "thumbnail", 
+              "thumbnail",
               "handle",
             ],
           },
           primaryKey: "id",
+          transformer: (product) => ({
```
Comment (shahednasser) [inline www/apps/docs/content/plugins/search/meilisearch.md:147]: The `transformer` isn't required, did you have to provide it for the plugin to work?

Comment (pavlotsyhanok) [reply www/apps/docs/content/plugins/search/meilisearch.md:147]: Hi @shahednasser yes, I had to provide it, otherwise it wouldn't work.

**Review by shahednasser — APPROVED** [review]
Thanks @pavlotsyhanok for this contribution! Can I get you to open an issue with more details on why you had to use the `transformer`?

**Comment by pavlotsyhanok** [comment]
> Thanks @pavlotsyhanok for this contribution! Can I get you to open an issue with more details on why you had to use the `transformer`?

Yes, will do.

**Comment by SalahAdDin** [comment]
> Problem:
> 
> The existing documentation for the Meilisearch plugin contains example that, if copy-pasted exactly, will not work.
> 
> Solution:
> 
> I have updated the Meilisearch plugin configuration example to reflect the correct settings.
> 
> Benefits:
> 
> With this updated code snippet, users can simply copy and paste it into their local medusa-config.js file. It will work seamlessly with the standard configuration and is compatible with the latest version of Meilisearch.
> 
> Testing:
> 
> The updated configuration has been tested and confirmed to work in a live deployment environment on Digital Ocean(Server) and Vercel(Storefront)
> image image

which medusa version is that for?

**Pull request merged**